source/index.js

```diff
@@ -21,11 +21,14 @@ try {
   throw new Error('Failed to load Sequelize. Have you installed it? Run `npm install sequelize`');
 }
 
-const { Sequelize, DataTypes, QueryTypes } = require('sequelize');
+const { Sequelize, DataTypes } = require('sequelize');
 
 // Ensure Sequelize version compatibility.
 const semver = require('semver');
-const sequelizeVersion = require('sequelize/package.json').version;
+const { version, release } = require('sequelize/package.json');
+const branchVersion = release.branches ? release.branches[0] : release.branch;
```

Comment:
Nit: can you add a comment here explaining the logic, why we need this check for release.branch vs release.branches